superset/views/core.py

```diff
@@ -3247,9 +3281,9 @@ def _insert_geocoded_data(
                 where = "WHERE " + where_clause % (row[:number_of_columns])
                 connection.execute(text(update + where))
             transaction.commit()
-        except Exception as e:
+        except Exception:
             transaction.rollback()
-            raise e
+            raise Exception("An error occured while inserting geocoded addresses")
```

Comment:
Could we raise a more specific Exception here? so that we can catch this with the Exception-type and return a specific message to the user?